app/models/invite.rb

```diff
@@ -257,10 +266,6 @@ def ensure_max_redemptions_allowed
 
   def ensure_no_invalid_email_invites
```

Comment:
This method and the associated validation can be removed as it no longer does anything.